src/Analysis/Abstract/Caching/FlowInsensitive.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving, TypeOperators #-}
+{-# LANGUAGE DeriveFunctor, FlexibleContexts, GeneralizedNewtypeDeriving, TypeApplications, TypeOperators #-}
```

Comment:
A lot of these `FlexibleContexts` additions are going to go away with `fused-effects` 1.0.